clients/python/setup.py

```diff
@@ -5,12 +5,14 @@
 
 setuptools.setup(name='myanmartools',
     version='0.1',
+    url='https://github.com/google/myanmar-tools',
     author='William (Wai Yan) Zhu',
     author_email='williamzhu345@gmail.com',
+    license='Apache License, Version 2.0',
     description='Tools for processing font encodings used in Myanmar',
     long_description=long_description,
     long_description_content_type='text/x-rst',
```

Comment:
Add some keywords?  "zawgyi" is not mentioned here at all and seems like a likely keyword.

Reply:
Yes, you can add
```python
keywords=[
    'burmese',
    'encoding',
    'myanmar',
    'myanmar unicode',
    'natural language processing',
    'nlp',
    'zawgyi'
]
```
However, I cannot find any source of information on how they are useful.

Reply:
Great; can you add them here to this PR?